scripts/frr/configs/commands/policy.json

```diff
@@ -36,6 +36,7 @@
     "/policy/route/route-map/@element/rule/@element/set/ip-next-hop":"set ip next-hop {/@text}",
     "/policy/route/route-map/@element/rule/@element/set/weight":"set weight {/@text}",
     "/policy/route/route-map/@element/rule/@element/set/metric":"set metric {/@text}",
+    "/policy/route/route-map/@element/rule/@element/set/src":"set src {/@text}",
```

Comment:
Please could you explain the reason for this change, I don't see a `src` node in the YANG module?

Reply:
Nevermind, I see your DANOS/vyatta-protocols-common#1 PR now! :-)